scripts/pi-hole/js/groups-clients.js

```diff
@@ -100,6 +100,7 @@ function initTable() {
       dataSrc: "clients",
       type: "GET",
     },
+    pagingTag: "button",
```

Comment:
After adding the bootstrap3 integration, this seems to be unnecessary.

Reply:
I reverted the commit.